lib/spack/spack/modules.py

```diff
@@ -428,13 +428,23 @@ def header(self):
     def module_specific_content(self, configuration):
         return tuple()
 
+    def verbose_autoload(self):
```

Comment:
This shouldn't be a classmethod, it does not use self.  Can you move it up out of the class, to right below the definition of `_module_config`?